proto/addressmetadata.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package models;
+
+message Header {
+    string Name = 1;
+    string Value = 2;
+}
+
+// Row is an indidual piece of structured data provided by wallet authors.
+// The headers help wallets determine how to deserialize the data for their purposes.
+message MetadataField {
+	repeated Header Headers = 1;
```

Comment:
Do we even want to separate the headers as opposed to just storing everything as some arbitrary bytes?

Reply:
I think we should define it here so the downstream consumers can at least guaranteee there's some metadata they can parse before deciding what to do with the actual Values.

The alternative is to have a "by convention" way to specify what type of data is in the payload anyways.